modules/end-user-guide/pages/url-parameter-for-the-ide.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: CONCEPT
+:description: URL parameter for the IDE
+:keywords: ide, editor, workspace-ide, workspace-editor
+:navtitle: URL parameter for the IDE
+:page-aliases: url-parameter-for-the-in-browser-ide.adoc
+
+[id="url-parameter-for-the-ide"]
+= URL parameter for the IDE
+
+If the URL for starting a new workspace doesn't contain a parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE, which is Microsoft Visual Studio Code - Open Source.
+
+The URL parameter for specifying a supported IDE is `che-editor=`. You can use it to start a workspace with a clone of the same project repository and your choice of a supported IDE.
+
+TIP: You can use the `che-editor=` parameter when you cannot add or edit a xref:defining-a-common-ide.adoc[`/.che/che-editor.yaml`] file in the source-code Git repository to be cloned for workspaces.
+
+NOTE: If the source-code Git repository to be cloned for workspaces contains a xref:defining-a-common-ide.adoc[`/.che/che-editor.yaml`] file, then the `che-editor=` parameter overrides it.
```

Comment:
```suggestion
NOTE: If the source-code Git repository to be cloned for workspaces contains a xref:defining-a-common-ide.adoc[`/.che/che-editor.yaml`] file, the `che-editor=` parameter overrides it.
```

Reply:
Thank you. I usually add `then` for better readability.